docs/source/algorithms/IndirectDiffScan-v1.rst

```diff
@@ -0,0 +1,43 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+This algortithm is to speed up the workflow for Indirect Inelastic diffraction reduction.
+Uses :ref:`ISISIndirectDiffractionReduction <algm-ISISIndirectDiffractionReduction>` in diffspec mode with several parameters at fixed values.
+Also extracts the temperature from each run and creates a workspace of all runs with respect to temperature.
+
+Usage
+-----
+
+**Example - Running IndirectDiffScan.**
+
+.. testcode:: ExIndirectDiffScan
+
+    ISISIndirectDiffractionReduction(InputFiles='IRS21360.raw',
+                                     OutputWorkspace='DiffractionReductions',
+                                     Instrument='IRIS',
+                                     SpectraRange=[105,112])
+
+    ws = mtd['DiffractionReductions'].getItem(0)
+
+    print 'Workspace name: %s' % ws.name()
+    print 'Number of spectra: %d' % ws.getNumberHistograms()
+    print 'Number of bins: %s' % ws.blocksize()
```

Comment:
Would it make sense to use the `print()` function in the examples? The users may need to move on to Python 3 at some point.